Merge new functions (#4)
fcrespel/runtastic-export-converter: README.md

    java -jar runtastic-export-converter-jar-with-dependencies.jar
    
    Expected arguments:
      check   <export path>
      list    <export path> <filter>
      user    <export path>
      info    <export path> <activity id>
      photo   <export path> <photo id>
      convert <export path> <activity id | 'all'> <destination path> ['gpx' | 'tcx']
      overlap <export path> <destination path> ['gpx' | 'tcx']
      help

To check an export (here, `runtastic-export-20190807-000` in the current directory):

    java -jar runtastic-export-converter-jar-with-dependencies.jar list runtastic-export-20190807-000

To list sport sessions from an export directory (here, `runtastic-export-20190807-000` in the current directory):

    java -jar runtastic-export-converter-jar-with-dependencies.jar list runtastic-export-20190807-000

> To list sport sessions from an export directory, for a "specific" filter, add the filter (keyword) as 3rd option. The tool searches within session description, session id, photo id, photo description and equipment id for the filter (keword). Here an example seraching for 'Marathon':

Reviewer: Typos:
```suggestion
To list sport sessions from an export directory, for a "specific" filter, add the filter (keyword) as 3rd option. The tool searches within session description, session id, photo id, photo description and equipment id for the filter (keyword). Here an example searching for 'Marathon':

```

Author: Thanks, fixed.

    
    Expected arguments:
      list <export path>
      user <export path>
      info <export path> <activity id>

>       photo <exort path> <photo id>

Reviewer: Typo: exort => export
```suggestion
      photo <export path> <photo id>
```

    java -jar runtastic-export-converter-jar-with-dependencies.jar convert runtastic-export-20190807-000 all runtastic-export-gpx gpx

To calculate overlapping sport sessions:

    java -jar runtastic-export-converter-jar-with-dependencies.jar overlap runtastic-export-20190807-000 

The command above analyzes all sport sessions and searches for 'overlapping' sport sessions bounds on the bounds of each sport session. 

To calculate overlapping sport sessions and export them:

    java -jar runtastic-export-converter-jar-with-dependencies.jar overlap runtastic-export-20190807-000 runtastic-export-gpx gpx

The command above analyzes all sport sessions and searches for 'overlapping' sport sessions bounds on the bounds of each sport session.

> It further exports them into destniation path. The exported sport session contains all overlapping sport sessions. 

Reviewer: Typo:
```suggestion
It further exports them into destination path. The exported sport session contains all overlapping sport sessions. 
```

Author: Thanks, fixed!
